reference/5.1/Microsoft.PowerShell.Core/About/about_Calculated_Properties.md

```diff
@@ -0,0 +1,518 @@
+---
+Locale: en-US
+ms.date: 08/07/2020
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_calculated_properties?view=powershell-5.1&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about_Calculated_Properties
+---
+# About calculated properties
+
+## Short Description
+
+PowerShell provides the ability to dynamically add new properties and alter
+the formatting of objects output to the pipeline.
+
+## Long Description
+
+A number of PowerShell cmdlets transform, aggregate, or process input objects
+into output objects using parameters that allow the addition of new properties
+to those output objects. These parameters can be used to generate new,
+calculated properties on output objects based on the values of input objects.
+The calculated property is defined by a [hashtable](about_hash_tables.md)
+containing key-value pairs that specify the name of the new property, an
+expression to calculate the value, and optional formatting information.
+
+## Supported cmdlets
+
+The following cmdlets support calculated property values for the **Property**
+parameter. The `Format-*` cmdlets also support calculated values for the
+**GroupBy** parameter.
+
+The following list itemizes the cmdlets that support calculated properties and
+the key-value pairs that are supported by each cmdlet.
+
+- `Compare-Object`
+  - `expression`
+
+- `ConvertTo-Html`
+  - `name`/`label` - optional (added in PowerShell 6.x)
+  - `expression`
+  - `width` - optional
+  - `alignment` - optional
+
+- `Format-Custom`
+  - `expression`
+  - `depth` - optional
+
+- `Format-List`
+  - `name`/`label` - optional
+  - `expression`
+  - `formatstring` - optional
+
+  This same set of key-value pairs also apply to calculated property values
+  passed to the **GroupBy** parameter for all `Format-*` cmdlets.
+
+- `Format-Table`
+  - `name`/`label` - optional
+  - `expression`
+  - `formatstring` - optional
+  - `width` - optional
+  - `alignment` - optional
+
+- `Format-Wide`
+  - `expression`
+  - `formatstring` - optional
+
+- `Group-Object`
+  - `expression`
+
+- `Measure-Object`
+  - Only supports a script block for the expression, not a hashtable.
+  - Not supported in PowerShell 5.1 and older.
+
+- `Select-Object`
+  - `name`/`label` - optional
+  - `expression`
+
+- `Sort-Object`
+  - `expression`
+  - `ascending`/`descending` - optional
+
+> [!NOTE]
+> The value of the `expression` can be a script block instead of a
+> hashtable. For more information, see the [Notes](#notes) section.
```

Comment:
Does this mean the `expression` field of the hashtable, or does it mean a scriptblock can be used as the whole value passed into the parameter?

Reply:
A scriptblock can be used as the whole value passed into the parameter. In fact, that is the required format for `Measure-Object`.

Reply:
Ok -- because of the use of `expression` it reads to me as the opposite, because `expression` has been referring to a field in hashtable inputs up to this point